rclcpp/QUALITY_DECLARATION.md

```diff
@@ -2,7 +2,7 @@ This document is a declaration of software quality for the `rclcpp` package, bas
 
 # `rclcpp` Quality Declaration
 
-The package `rclcpp` claims to be in the **Quality Level 4** category.
+The package `rclcpp` claims to be in the **Quality Level 2** category.
```

Comment:
All these packages have level 4 dependencies, so they can't be bumped to level 2 until those are bumped.